src/bundle/Resources/public/scss/_contentbox.scss

```diff
@@ -0,0 +1,77 @@
+.ez-contentbox {
+    width: 100%;
+
+    .ez-list-toolbar {
+        display: flex;
+        align-items: center;
+        padding: 0.5em 0;
+        background: $ez-color-base-pale;
+        border-radius: 4px 4px 0 0;
+        font-size: 1rem;
+
+        .ez-list-toolbar-label {
+            flex: 1;
+            margin: 0;
+            padding: 0 22px;
+            font-size: 1.1em;
+            line-height: 2;
+            font-weight: bold;
+        }
+    }
+}
+
+.contentbox-body {
```

Comment:
I think is should be `.ez-contentbox__body` and you can move it inside `.ez-contentbox` and write: `&__body`.